Packs/ArcSightLogger/Integrations/integration-ArcSightLogger.yml

```diff
@@ -647,7 +647,7 @@ script:
       predefined:
       - "true"
       - "false"
-      description: Indicates the search is local only, and does not include peers. The default value for this parameter is 'true'.
+      description: Indicates the search is local only, and does not include peers. The default value for this parameter is 'true'. If the local search is configured to 'False' in your instance, make sure to add this argument with the 'false' value to the command.
```

Comment:
I think you need to clarify which instace you are referring here.
Youre talking about the ArcSight Logger right? not the XSOAR instance of it.